hudi-common/src/main/java/org/apache/hudi/metadata/HoodieTableMetadataUtil.java

```diff
@@ -955,8 +956,9 @@ public static HoodieTableFileSystemView getFileSystemView(HoodieTableMetaClient
     // though we may have initialized them.
     HoodieTimeline timeline = metaClient.getActiveTimeline();
     if (timeline.empty()) {
+      HoodieTimeGeneratorConfig timeGeneratorConfig = HoodieTimeGeneratorConfig.newBuilder().withPath(metaClient.getBasePathV2().toString()).build();
       final HoodieInstant instant = new HoodieInstant(false, HoodieTimeline.DELTA_COMMIT_ACTION,
```

Comment:
The write config/lock provider config does not pass around correctly, not sure whether we should take a lock here as this is a metadata table.

Reply:
This method is just an initialize step, right? If so, I think the instantTime generated here is just a fake time, and won't be used actually.